libr/cons/utf8.c

```diff
@@ -191,40 +191,47 @@ static int cursor_position(const int tty, int *const rowptr, int *const colptr)
 }
 #endif
 
-R_API int r_cons_is_utf8() {
-	int ret = 0;
+R_API bool r_cons_is_utf8() {
+	bool ret = false;
 #if UTF8_DETECT_ENV
 	char *sval = r_sys_getenv ("LC_CTYPE");
 	if (sval) {
 		r_str_case (sval, 0);
 		if (!strcmp (sval, "utf-8")) {
-			ret = 1;
+			ret = true;
 		}
 		free (sval);
 	}
+	char *lang = r_sys_getenv ("LANG");
```

Comment:
this conflicts with scr.utf8 when spawning a new process. this is why the tests are failing